sdk/core/azure-core-tracing-opentelemetry/test/ut/service_support_test.cpp

```diff
@@ -333,17 +346,15 @@ TEST_F(OpenTelemetryServiceTests, CreateWithImplicitProvider)
     auto provider(
         Azure::Core::Tracing::OpenTelemetry::OpenTelemetryProvider::Create(tracerProvider));
 
-    Azure::Core::Context::ApplicationContext.SetTracerProvider(provider);
-
     {
       Azure::Core::_internal::ClientOptions clientOptions;
       clientOptions.Telemetry.ApplicationId = "MyApplication";
+      clientOptions.Telemetry.TracingProvider = provider;
 
       Azure::Core::Tracing::_internal::TracingContextFactory serviceTrace(
           clientOptions, "my-service", "1.0beta-2");
```

Comment:
Version here is an arbitrary string (because this is test collateral). It doesn't need to be a semver.

If you're going to be pedantic here, the service name "my-service" isn't a valid Azure Resource Manager service either, but the service name in the API is required to be a valid ARM service name.